embedly/embedly.py

```diff
@@ -54,13 +52,21 @@ def get_urls_from_embedly(urls):
     except requests.RequestException:
         response = None
 
+    embedly_data = []
     if response is not None:
         try:
-            response_data = json.loads(response.content)
+            embedly_data = json.loads(response.content)
         except (TypeError, ValueError):
             pass
 
-    return {url_data['original_url']: url_data for url_data in response_data}
+    parsed_data = {}
+    if type(embedly_data) is list:
+        parsed_data = {
+            url_data['original_url']: url_data
+            for url_data in embedly_data
+        }
+
+    return parsed_data
```

Comment:
I think if proxy fails to fetch data from embedly for whatever reason, it should just return an error to the client rather than sending back a 200 with an empty payload. Also, it shouldn't cache this in Redis.